CMakeLists.txt

```diff
@@ -2,6 +2,9 @@ cmake_minimum_required(VERSION 3.3)
 
 set(LLVM_SPIRV_VERSION 0.2.1.0)
 
+option(LLVM_SPIRV_INCLUDE_TESTS
```

Comment:
I think it would be perfect if this option (unless explicitly specified) followed `LLVM_INCLUDE_TESTS`.

I.e. tests for the translator are enabled or disabled by default based on LLVM settings, but this can be specifically configured if user don't want to follow LLVM settings.

@svenvh, if this is not so easy to do, or you don't have enough time, I'm good with this patch as-is, thanks!

Reply:
I can change the default initialization to `LLVM_INCLUDE_TESTS` instead of `ON`, if that is preferable?

Reply:
> I can change the default initialization to `LLVM_INCLUDE_TESTS` instead of `ON`, if that is preferable?

If it works it would be great, because it basically will preserve backward compatibility: right now translator follows `LLVM_INCLUDE_TESTS` and this patch breaks this behavior

Reply:
Defaulting to `LLVM_INCLUDE_TESTS` seems to be what `CLANG_INCLUDE_TESTS` and `LLDB_INCLUDE_TESTS` do as well, so it makes sense. Thanks for the suggestion!